string itemName
uint8 dxlID
---
float64 getData